cognite/neat/_constants.py

```diff
@@ -193,6 +193,8 @@ def get_default_prefixes_and_namespaces() -> dict[str, Namespace]:
     "view_property": "path",
 }
 
+BASE_MODEL = Literal["CogniteCore"]
```

Comment:
Nit: I believe Python naming convention states that user-defined types (including aliases) should be like `BaseModel`?